tasks.py

```diff
@@ -24,17 +24,28 @@
 ns = Collection()
 
 build_tasks = Collection()
-ns.add_collection(build_tasks, name='build')
+ns.add_collection(build_tasks, name="build")
 
 
 @task
 def build(ctx):
-    ctx.run('python -m build')
+    ctx.run("python -m build")
 
-build_tasks.add_task(build, default=True, name='build')
+
+build_tasks.add_task(build, default=True, name="build")
+
+
+@task
+def release_build(ctx):
+    ctx.run('pip install "$(grep build setup.cfg | cut -f2 -d-)"')
```

Comment:
pip should be invoked as `python -m pip` rather than just `pip`, which is considered safer (even if, on a properly configured host, it should be the same)